src/main/java/io/vlingo/http/resource/RequestHandler2.java

```diff
@@ -16,6 +16,7 @@
 import io.vlingo.http.Response;
 
 import java.util.Arrays;
+import java.util.Map;
```

Comment:
I think it has been imported by accident

Reply:
Yes,  must have been a mistake.  Thanks for catching that! This was a low-effort change, but good introduction to the module.